packages/docs/src/styles/components/_ReactPropDocs.scss

```diff
@@ -0,0 +1,63 @@
+// Responsive docs table
+
+.docs_table--wrapper {
+  overflow-x: auto;
+}
+
+.docs_table {
+  @media (max-width: $width-sm) {
+    table,
+    thead,
+    tbody,
+    th,
+    td,
+    tr {
+      display: block;
```

Comment:
Thanks @nichia for jumping on a call this morning to talk about testing with screen readers, and for being proactive about accessibility. 🎉 

This solution looks elegant, but unfortunately breaks the semantics of tables for screen readers. Using `display: block` on tables elements means that screen readers don't know it's a table any longer. So users can't get the extra context needed when browsing it.

I shared these links with @nichia, which outline why and how to approach responsive tables in an accessible way. It's more code, but possible.

See:
What display properties do to tables: https://adrianroselli.com/2018/02/tables-css-display-properties-and-aria.html
A responsive, accessible table: https://adrianroselli.com/2017/11/a-responsive-accessible-table.html

I'd recommend taking one of the approaches outlined in the post. I'd be happy to test again too!